### PR TITLE
Error message made red colour GCI task issue #548

### PR DESCRIPTION
**Pull request by mohit-003**
Fixes #
Fixed The color of the text. Made it Red to be notisable.
#### Describe the changes you have made in this pr -
Changed the styles of "#error_messages"
### Screenshots of the changes (If any) -
aaaa

### CODE REVIEW

**Comment by ankitkataria** [comment]
Great work @mohit-003 !
Could we reduce the font-size of the error heading? It should be smaller than the main heading (i.e. "Sign Up"). Screenshots for the change will be appreciated. :grin:

File: public/css/logixRelease.css [hunk 1]
```diff
@@ -109,4 +109,10 @@ a.btn.btn-outline-danger {
     margin-top: 10px;
 }
 
+#error_explanation{
+    color: crimson;
+    background: #cea0a5;
```
Comment (alanjsph) [inline public/css/logixRelease.css]: Do we really need this `backgroud` property set? What happens if you remove it?

File: public/css/logixRelease.css [hunk 2]
```diff
@@ -109,4 +109,10 @@ a.btn.btn-outline-danger {
...
+    border: 1px solid #86181d;
+    padding: 4px;
+    }
```
Comment (alanjsph) [inline public/css/logixRelease.css]:
```suggestion
}
```

**Comment by alanjsph** [comment]
Also, like Ankit has mentioned, can we make this error message a bit subtle ?
Something like - https://stackoverflow.com/a/15156538

File: public/css/logixRelease.css (hunk 1 above)
Comment (mohit-003) [reply public/css/logixRelease.css]: I thought #f0f0f0 may be good
What do you suggest?

**Comment by mohit-003** [comment]
I have changed the size of the text. Is the background colour good? Or should I change it?

**Comment by mohit-003** [comment]
> Great work @mohit-003 !
> Could we reduce the font-size of the error heading? It should be smaller than the main heading (i.e. "Sign Up"). Screenshots for the change will be appreciated.

Yep done

**Comment by Shweta0002** [comment]
> I have changed the size of the text. Is the background colour good? Or should I change it?
No ,It seems good

**Pull request merged**